src/IntegrationTests/IntegrationTests/OpenTelemetryTraceExporterSmokeTest.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Newtonsoft.Json;
+using IntegrationTests.Fixtures;
+using System.Net.Http;
+using System.Threading;
+using Xunit;
+using Xunit.Abstractions;
+using System.Collections.Generic;
+using System.Linq;
+using System;
+
+namespace IntegrationTests
+{
+    public class OpenTelemetryTraceExporterSmokeTest : IClassFixture<OpenTelemetryUsageApplicationFixture>
+    {
+        private readonly OpenTelemetryUsageApplicationFixture _fixture;
+
+        private readonly string _insightsQueryApiKey;
+
+        private readonly string _insightsQueryApiEndpoint = "https://staging-insights-api.newrelic.com";
```

Comment:
If a customer runs this with their API key set up in environment variables....   Will it work to send the data to the staging endpoint?

Reply:
it is a wrong endpoint!!!